### PR TITLE
Add ability to provide additional rpmbuild-options to tito

### PR DESCRIPTION
**Pull request by arilivigni**
- Add SRPM generation for CBS
- mv srpm from container location to host location

I did a scratch build in CBS based on output of src.rpm generated
https://cbs.centos.org/koji/taskinfo?taskID=178616

### CODE REVIEW

**Comment by arilivigni** [comment]
[test]

File: hack/build-rpm-release.sh [hunk 1]
```diff
@@ -23,32 +23,33 @@ tito tag --use-version="${OS_RPM_VERSION}" \
...
+tito build --offline --srpm --rpmbuild-options="--define 'dist .el7'" --output="${tito_tmp_dir}"
 tito build --output="${tito_tmp_dir}" --rpm --no-cleanup --quiet --offline \
-           --rpmbuild-options="--define 'make_redistributable ${make_redistributable}'"
+           --rpmbuild-options="--define 'make_redistributable ${make_redistributable}' ${RPM_BUILD_OPTS:-}"
 tito tag --undo --offline
 
 os::log::info 'Unpacking tito artifacts for reuse...'
 output_directories=( $( find "${tito_tmp_dir}" -type d -name "rpmbuild-${OS_RPM_NAME}*" ) )
 if [[ "${#output_directories[@]}" -eq 0 ]]; then
-	os::log::error 'After the tito build, no rpmbuild directory was found!'
```
Comment (stevekuznetsov) [inline hack/build-rpm-release.sh]: Can we put the formatting in a separate commit or omit it, please?

File: hack/build-rpm-release.sh [hunk 2]
```diff
@@ -23,32 +23,33 @@ tito tag --use-version="${OS_RPM_VERSION}" \
          --no-auto-changelog --offline
 tito_tmp_dir="${BASETMPDIR}/tito"
 mkdir -p "${tito_tmp_dir}"
+tito build --offline --srpm --rpmbuild-options="--define 'dist .el7'" --output="${tito_tmp_dir}"
```
Comment (stevekuznetsov) [inline hack/build-rpm-release.sh]: I'm not familiar with SRPM too much -- but why does the distribution matter for what should be a compressed bundle of source files?

**Comment by stevekuznetsov** [comment]
[test]

**Comment by stevekuznetsov** [comment]
[merge]

**Comment by stevekuznetsov** [comment]
@openshift-bot, the [last build](https://ci.openshift.redhat.com/jenkins/job/merge_pull_request_origin/565/) failed from the following flakes:
 - [conformance](https://ci.openshift.redhat.com/jenkins/job/merge_pull_request_origin/565/testReport/junit/(root)/Extended/_k8s_io__StatefulSet__k8s_io__Basic_StatefulSet_functionality__StatefulSetBasic__Scaling_down_before_scale_up_is_finished_should_wait_until_current_pod_will_be_running_and_ready_before_it_will_be_removed/) failed on: #13106
